modules/onetagBidAdapter.js

```diff
@@ -6,13 +6,13 @@ const ENDPOINT = 'https://onetag-sys.com/prebid-request';
 const USER_SYNC_ENDPOINT = 'https://onetag-sys.com/usync/';
 const BIDDER_CODE = 'onetag';
 const BANNER = 'banner';
+const VIDEO = 'video';
 
 // =======
```

Comment:
Are these comment lines necessary?

Reply:
Hello @Fawke , we've just updated the pull request following your suggestions and added the test parameters for the video case